### ISSUE TITLE
Problem with loading of the citation

### ISSUE THREAD

**Issue by ChrisSimeon** [deanishe/zothero#43]
Hey, 

I don't know if this is normal or not but or me it takes some time to load the citation into the clipboard. It's kind of strange, everything is fine, i hit command+ enter to copy the citation then I want to fill it in but when I hit command + V it pastes whatever was in my clipboard before. I have to wait 2 seconds or sth like that until it is loaded in the clipboard end then I can paste with command+v. 
Is this normal? Is there anything I can do to make it faster? It's not the biggest problem but it breaks my workflow a little bit to wait like this. 

But thanks alot for your work, it still is way faster than opening the program.

**Comment by deanishe** [comment]
> Is this normal?

No. It can take a second or so to generate the citation, but Alfred's window should remain open during that time. The citation should be on the clipboard as soon as Alfred's window disappears.

Which application(s) are you pasting in? Do you have the problem in all applications? Are you running any clipboard history applications?

**Comment by ChrisSimeon** [comment]
Thanks for your anwer! I tried it with a few applications and it is the same everywhere. Since I have Alfred I only use the integrated clipboard history function, but I had another one installed before, which was still installed when I set up Alfred and Zothero could it be that it wasn't completely deleted? Is it possible to "re-install" the clipboard function to make sure?

**Comment by ChrisSimeon** [comment]
Ok update. I still haven't found the problem exactly but it seems like the problem is anywhere near the shell. I also changed the shell to bash but hasn't changed anything. I have no idea what zsh and bash could have in common that takes so long to run the script. I now pushed the "hide alfred" function after the script so at least I don't paste in the wrong content from the clipboard. So it is in fact not a problem with your workflow, but do you maybe have an idea what it could be?

**Comment by deanishe** [comment]
> but do you maybe have an idea what it could be?

You're not giving me much information to work with, and I'm not a shell expert by any stretch.

When a shell is slow to start, it's usually because of your shell configuration files. If both shells are slow, the obvious answer would be that something you've installed has added slow code to both of their config files.

zsh is easier to debug because the only file that should be loaded in a non-interactive shell (i.e. the workflow) is ~/.zshenv. Check what's in that file.
